devel/build

```
@@ -19,7 +19,14 @@ export GIT_REVISION=$(git describe --tags --abbrev=40 --always --dirty || true)
 # Calling `docker run nextstrain/base` will still only pull down the small base
 # image rather than pulling down the larger nextstrain/base-builder image.
 
-docker build \
+platform=linux/amd64
+
+# `buildx create` is necessary to use a driver that supports multi-platform
+# images.
+builder=$(docker buildx create --use)
```

Since we're capturing the builder name here, we don't need `--use`. And it's nice not to make a temporary global state change on the user, as that could have other impacts.

```suggestion
builder=$(docker buildx create)
```

---

Done: e70bf410aa936a83a5330a89af4570ccb807cf77